gn3/settings.py

```diff
@@ -27,3 +27,21 @@
 
 # wgcna script
 WGCNA_RSCRIPT = "wgcna_analysis.R"
+# qtlreaper command
+REAPER_COMMAND = "{}/bin/qtlreaper".format(os.environ.get("GUIX_ENVIRONMENT"))
+
+# genotype files
+GENOTYPE_FILES = os.environ.get(
+    "GENOTYPE_FILES", "{}/genotype_files/genotype".format(os.environ.get("HOME")))
```

Comment:
**suggestion:** I think there's is a variable for this: `GENENETWORK_FILES`: so this would be:

```suggestion
GENOTYPE_FILES = os.path.join(
    os.environ.get("GENENETWORK_FILES",
                   f"{os.environment.get('HOME')}/genotype_files")
    "genotype")
    "GENENETWORK_FILES", "{}/genotype_files/genotype".format(os.environ.get("HOME"))) + "/j
```